native/cocos/editor-support/spine-creator-support/SkeletonRenderer.h

```diff
@@ -38,17 +38,17 @@
 #include "base/RefMap.h"
 #include "core/assets/Texture2D.h"
 #include "middleware-adapter.h"
+#if CC_USE_SPINE_3_8
 #include "spine-creator-support/VertexEffectDelegate.h"
+#endif
 #include "spine/spine.h"
 
 namespace cc {
 class RenderEntity;
 class RenderDrawInfo;
 class Material;
-}; // namespace cc
-
-namespace spine {
 
+using namespace spine;
```

Comment:
Don't use `using namespace spine;` in header file which may make other file that include `SkeletonRenderer.h` also use `spine` namespace.

Reply:
Fixed